### DISCUSSION TITLE
Budibase relationships postgress let data disappear

### DISCUSSION THREAD

**Discussion by JeroenP1974** [Budibase/budibase#13242] [Help]
I have two tables :

1) Groepen 

<img width="457" alt="Scherm­afbeelding 2024-03-12 om 22 12 01" src="https://github.com/Budibase/budibase/assets/157612513/d2d53d7b-51a1-4eb3-b71f-3adaf8e6194f">

2) Producten

<img width="1335" alt="Scherm­afbeelding 2024-03-12 om 22 12 20" src="https://github.com/Budibase/budibase/assets/157612513/32f40f6a-58ee-4cef-a6fd-5c4b9a75afaf">

I setup the relationships as follows :

<img width="597" alt="Scherm­afbeelding 2024-03-12 om 22 12 51" src="https://github.com/Budibase/budibase/assets/157612513/65f53163-1ac4-4324-a2d9-d071e0969fd7">

When I go to the table 'Groepen' there is data missing :

<img width="651" alt="Scherm­afbeelding 2024-03-12 om 22 13 52" src="https://github.com/Budibase/budibase/assets/157612513/41fa2004-015e-48a9-8acb-bf1cd1979847">

And also when I go to the form there is no possibility to select (dropdown) a new group. 

<img width="643" alt="Scherm­afbeelding 2024-03-12 om 22 15 18" src="https://github.com/Budibase/budibase/assets/157612513/be6b065d-33ce-407c-bab1-e93a8fb45a70">

Could really use some help here! Thank you all for your answers and help.

**Comment by ConorWebb96** [comment]
Hey @JeroenP1974,

Not seeing this behaviour on my version of Budibase which is on the lastest version. Possibly update to the latest version of Budibase and see if that resolves your issues.

Make sure your relationships on your database are setup correctly. E.g. Correct primary and foreign keys linked on the tables.
